sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -24,82 +73,31 @@ class UniversalSpinTable(Base):
     repointing_number = Column(Integer, nullable=False)
 
 
-class FileCatalogTable:
-    """Common file catalog table"""
+class StatusTrackingTable(Base):
+    """Status tracking table"""
+
+    __tablename__ = "status_tracking_table"
```

Comment:
```suggestion
class StatusTracking(Base):
    """Status tracking table"""

    __tablename__ = "status_tracking"
```

I'm not sure we need the word "table" in these names. In my mind, it can be assumed we are talking about tables since this is for a database.